src/configuration/security/security-txt.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Security.txt
+---
+
+Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Security]({% link configuration/security.md %}) > Security.txt
+
+## General
+
+![]({% link images/images/config-security-txt-general.png %}){: .zoom}
+[_General_]({% link stores/security-issue-reporting.md %})
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enable|Website|When enabled, a `security.txt` file is saved that contains information that is needed by security researchers to report potential vulnerabilities to you. Options:<br /> Yes - Creates the `security.txt` file based on information entered in the `Contact information` and `Other information` sections.<br />No - (default) Does not create the `security.txt` file.|
+
+## Contact information
+
+![]({% link images/images/config-security-txt-contact-info.png %}){: .zoom}
+[_Contact Information_]({% link stores/security-issue-reporting.md %})
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Email|Website|The email address where security reports can be sent.|
+|Phone|Website|A phone number that can be used to report security concerns.|
+|Contact Page|Website|The URL of a page on your site that lists security contacts, or your _Contact Us_ page. Examples: `https://mystore.com/security-contact.html`or `https://mystore.com/contact/`|
+
+## Other information
+
+![]({% link images/images/config-security-txt-other-info.png %}){: .zoom}
+[_Other Information_]({% link stores/security-issue-reporting.md %})
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Encryption|Website|A URL that points to the location of an encryption key that security researchers can use to send encrypted communications. _Do not enter the encryption key in this field._ It is the responsibility of the researcher to verify that the key is from a trustworthy source. Researchers must not assume that the key is the same as that used to generate the digital signature. Examples:<br /> OpenPGP key from web server - `https://mystore.com/pgp-key.txt`|
+|Acknowledgements|Website|A URL that points to a page in your store where security researchers are acknowledged. For example: `https://mystore.com/hall-of-fame.html` To prevent future attacks, include only a general description without revealing specific information about vulnerability issues. For example:<br />We would like to thank the following researchers:<br />(yyyy/mm/dd) Justin Thyme - SQL injection|
```

Comment:
```suggestion
|Acknowledgements|Website|A URL that points to a page in your store where security researchers are acknowledged, such as`https://mystore.com/hall-of-fame.html`. To prevent future attacks, include only a general description without revealing specific information about vulnerability issues. Example:<br />We would like to thank the following researchers:<br />(yyyy/mm/dd) Justin Thyme - SQL injection|
```